sparkcore/src/main/scala/quasar/physical/sparkcore/fs/hdfs/package.scala

```
@@ -175,7 +175,7 @@ package object hdfs {
     def fileSystem: Task[HdfsFileSystem] = Task.delay {
       val conf = new Configuration()
       conf.setBoolean("fs.hdfs.impl.disable.cache", true)
-      HdfsFileSystem.get(new URI(sparkFsConf.hdfsUriStr), conf)
```

Something to verify that a URI of that format decodes as expected. Further, was the error properly captured before or did it fail silently? If it was silent, it could be useful to detect and fail the Task.